### PR TITLE
{lang}[intel-2016a] Python 2.7.11 with libX11 1.6.3 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
Needs #2614

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-2.7.11-intel-2016a-libX11-1.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.9.2'),
+    (libx11, libxver),
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-intel-2016a-libX11-1.6.3.eb]: no need to include `libX11` separately?

File: easybuild/easyconfigs/p/Python/Python-2.7.11-intel-2016a-libX11-1.6.3.eb [hunk 2]
```diff
@@ -0,0 +1,139 @@
+name = 'Python'
+version = '2.7.11'
+
+libx11 = 'libX11'
+libxver = '1.6.3'
+versionsuffix = '-%s-%s' % (libx11, libxver)
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-intel-2016a-libX11-1.6.3.eb]: just make this

```python
versionsuffix = '-libX11-1.6.3`
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2151.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/bc2963f7df7343cd9d7f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2633.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ed4981864861e20e775e for a full test report.

**Comment by hajgato** [comment]
@boegel  Fixed

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**